shared/oae/api/oae.api.group.js

```diff
@@ -280,4 +280,32 @@ define(['exports', 'jquery', 'underscore', 'oae.api.util'], function(exports, $,
             }
         });
     };
+
+    /**
+     * Delete a group as the currently authenticated user.
```

Comment:
We can just say `Delete a group` here. The join and leave function just have the `as the currently authenticated user` bit as it makes more sense there